### ISSUE TITLE
Incompatible with latest version of Fabric API. getting invalid player data when using immersive portals

### ISSUE THREAD

**Issue by Ganju0** [qouteall/ImmersivePortalsMod#1652]
I'm not sure why I'm having this issue since I don't see any other issues like it on github. Using prism launcher, fresh install of 1.21.1. Only installing immersive portals with fabric. When creating a single player world I get the error "invalid player data"

I've tried deleting and making new instances of minecraft and it didn't help

**Comment by Ganju0** [comment]
[error logs.txt](https://github.com/user-attachments/files/17740475/error.logs.txt)

I closed this for a sec cause I realized I hadn't updated prism launcher. But after updating the issue persists. I've attached a log of what happens when I try to enter a world if it helps

**Comment by mars0124** [comment]
I was also running a fresh instance, etc.  Downgraded to the last version of Fabric API to resolve (Fabric API 0.107.0+)

**Comment by Kylecoolky** [comment]
I also have the same issue. Using Modrinth and the Fabric API. Happens even a fresh instance of MC 1.21.1 and Immersive Portals 6.0.4 on both old (on other instances) and new worlds.

**Comment by mars0124** [comment]
Did you downgrade Fabric API as described above? For me it was a Fabric issue, not an Immersive Portals issue.

**Comment by Kylecoolky** [comment]
> Did you downgrade Fabric API as described above? For me it was a Fabric issue, not an Immersive Portals issue.

Okay I swear I'm not an idiot I just didn't see the "to resolve" part of your comment and my brain kinda just skipped over it. It works now...

**Comment by mars0124** [comment]
No judgement. My brain skips texts constantly :) Glad you're up and running!

**Comment by qouteall** [comment]
```
Mixin apply for mod fabric-data-attachment-api-v1 failed fabric-data-attachment-api-v1.mixins.json:ChunkDataSenderMixin from mod fabric-data-attachment-api-v1 -> net.minecraft.class_8608: org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException @At("INVOKE") on net/minecraft/class_8608::sendInitialAttachmentData with priority 1000 cannot inject into net/minecraft/class_8608::method_52386(Lnet/minecraft/class_3222;)V merged by qouteall.imm_ptl.core.mixin.common.chunk_sync.MixinPlayerChunkSender with priority 1000 [PREINJECT Applicator Phase -> fabric-data-attachment-api-v1.mixins.json:ChunkDataSenderMixin from mod fabric-data-attachment-api-v1 -> Prepare Injections ->  -> wrapOperation$zbm000$fabric-data-attachment-api-v1$sendInitialAttachmentData(Lnet/minecraft/class_3244;Lnet/minecraft/class_3218;Lnet/minecraft/class_2818;Lcom/llamalad7/mixinextras/injector/wrapoperation/Operation;Lnet/minecraft/class_3222;)V -> Prepare]
org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException: @At("INVOKE") on net/minecraft/class_8608::sendInitialAttachmentData with priority 1000 cannot inject into net/minecraft/class_8608::method_52386(Lnet/minecraft/class_3222;)V merged by qouteall.imm_ptl.core.mixin.common.chunk_sync.MixinPlayerChunkSender with priority 1000 [PREINJECT Applicator Phase -> fabric-data-attachment-api-v1.mixins.json:ChunkDataSenderMixin from mod fabric-data-attachment-api-v1 -> Prepare Injections ->  -> wrapOperation$zbm000$fabric-data-attachment-api-v1$sendInitialAttachmentData(Lnet/minecraft/class_3244;Lnet/minecraft/class_3218;Lnet/minecraft/class_2818;Lcom/llamalad7/mixinextras/injector/wrapoperation/Operation;Lnet/minecraft/class_3222;)V -> Prepare]

```

**Comment by qouteall** [comment]
Probably fixed in 6.0.5 https://github.com/iPortalTeam/ImmersivePortalsMod/releases/tag/v6.0.5-mc1.21.1
